dev/NumberBox/TestUI/NumberBoxAxeTestPage.xaml.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Windows.UI.Xaml.Controls;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
```

Comment:
This comment probably should be removed.

Reply:
Yes, removed now.